lib/click-handler.js

```diff
@@ -110,19 +90,38 @@ function onClick(event) {
     return;
   }
 
-  tryLoad(urls, (err, url, res) => {
-    if (err) {
-      showTooltip($tooltipTarget, SORRY);
+  const track = action =>
+    chrome.runtime.sendMessage({
+      type: 'track',
+      payload: {
+        category: 'fetch',
+        action,
+      },
+    });
 
-      return console.error(err); // eslint-disable-line no-console
-    }
+  try {
+    // TODO remove this and fix tests
+    // Maybe use the following for mocking fetch?
+    // https://github.com/speedskater/babel-plugin-rewire
+    chrome.runtime.sendMessage({
+      type: 'fetch',
+      payload: urls,
+    });
```

Comment:
As mentioned in the comment, the tests currently stub out `sendMessage` and rely on it to be called. I briefly tried using https://github.com/speedskater/babel-plugin-rewire to do the same thing for our `fetch` module, but I wasn't able to get it working, so I just left things as they are for now.